lib/struct.js

```diff
@@ -237,7 +243,7 @@ module.exports = function() {
             var f = a[2];
             // we treat Z and z specially -- the length field is their
```

Comment:
Minor nit -- please fix the comment? Perhaps substitute 'Z and z' to say 'these values'